suite-native/app/src/navigation/RootStackNavigator.tsx

```diff
@@ -50,7 +50,7 @@ export const RootStackNavigator = () => {
             screenOptions={stackNavigationOptionsConfig}
         >
             <RootStack.Screen
-                name={RootStackRoutes.Onboarding}
+                name={RootStackRoutes.OnboardingStack}
                 component={OnboardingStackNavigator}
             />
```

Comment:
I believe this is missing:
```suggestion
                options={{ animation: 'slide_from_bottom' }}
            />
```

Reply:
68cc994372